caffe2/proto/caffe2.proto

```diff
@@ -11,27 +11,69 @@ package caffe2;
 //     is mainly for backward compability purposes but we may consider using
 //     those in the future.
 
+// ExternalDataProto stores the pointer to the content of TensorProto
+message ExternalDataProto {
+  // type of the external storage type, can be the following:
+  //   - FILE, external file, id is the path to the file
+  //   - TORCH_WRAPPER, the wrapper in torch/csrc/jit/serialization.h is used,
+  //     id is the tag
+  //   - DB, id is the serialized DBReaderProto
+  // NEW type to be ADDED here
+  optional string type = 1;
```

Comment:
What is the motivation for this to be string instead of enum?

Also as a second question - how access to the external files is supposed to be handled? Is it assuming thtat only POSIX files are supported for this path and everything else goes through DBReaderProto?

Also what is the plan to warranty integrity of the models for cases of production delivery? I can totally foresee cases when certain external files will be lost/not accessible for one or another reason.

Reply:
I agree, enum is nicer. Thanks :-)

Reply:
For the production delivery, we should only use DB or the wrapper. Everything should be in one file. External files are not recommend for the production. We probably can set the wrapper as our default option to avoid the problem.